frontend/device/kobo/device.lua

```diff
@@ -425,6 +425,7 @@ local KoboLuna = Kobo:new{
     hasFrontlight = yes,
     touch_phoenix_protocol = true,
     display_dpi = 212,
+    power_dev = "/dev/input/event2",
```

Comment:
Nope, potentially too brittle ;).

Check if it works with `automagic_sysfs = true,` (like the Libra 2) instead.

And if it doesn't, get me the output of `ls -lash /dev/input/by-path` ;).